src/cache/cache_client_builder.rs

```diff
@@ -11,6 +11,8 @@ use momento_protos::cache_client::scs_client::ScsClient;
 use momento_protos::control_client::scs_control_client::ScsControlClient;
 use tonic::transport::Channel;
 
+const DEFAULT_MAX_REQUEST_SIZE: usize = 10_486_000;
```

Comment:
Can we make this 20mib? 

Or is there a way to make this configurable by each client?